admin/admin.go

```diff
@@ -101,6 +101,23 @@ func (s *Server) AddVersionHandler(version string) {
 	})
 }
 
+// Subrouter creates and returns a subrouter with the specific prefix.
+//
+// The returned subrouter can Use() middleware without impacting
```

Comment:
I'm not sure what `can Use() middleware` refers to. Does this refer to our zero-trust middleware or something else?

Reply:
It's a method on mux.Router

Reply:
I can get rid of the function reference and just say `use` instead of `Use()`